src/components/SUE/report/SueReportLocation.tsx

```diff
@@ -162,6 +162,7 @@ export const SueReportLocation = ({
           name="street"
           label={texts.sue.report.street}
           placeholder={texts.sue.report.street}
+          textContentType="streetAddressLine1"
```

Comment:
are there also `streetAddressLine2`, `2streetAddressLine3` and other? what does they stand for?

Reply:
There are only streetAddressLine1 and streetAddressLine2. In the iOS system, it depends on what information the user saves. For us, streetAddressLine1 was necessary. This way we could get the most accurate data.

It can be examined in more detail from the document here.: https://reactnative.dev/docs/textinput#textcontenttype-ios